scripts/configure_valhalla.sh

```diff
@@ -219,12 +219,33 @@ if [[ "${do_build}" == "True" ]]; then
     fi
     maybe_create_dir ${ELEVATION_PATH}
     echo ""
-    echo "================================="
+    echo "================================"
     echo "= Download the elevation tiles ="
-    echo "================================="
+    echo "================================"
     valhalla_build_elevation --from-tiles --decompress -c ${CONFIG_FILE} -v || exit 1
   fi
 
+fi
+
+# Use OSMSpeeds default_speeds
+
+updated_default_speed_config=False
+if [[ $use_default_speeds_config == "True" ]]; then
+  if test -f "${DEFAULT_SPEEDS_CONFIG}"; then
+    echo "WARNING: found default speed config, skipping download"
+  else 
+
+    echo ""
+    echo "======================================"
+    echo "= Download the default speeds config ="
+    echo "======================================"
+    download_file "${default_speeds_config_url}" "${DEFAULT_SPEEDS_CONFIG}"
+    updated_default_speed_config=True
+  fi
```

Comment:
I wouldn't warn if there was already a json. that's a false alarm IMO. the graph build logs will tell the user if one was used or skipped.

```suggestion
  if ! test -f "${DEFAULT_SPEEDS_CONFIG}"; then
    echo ""
    echo "======================================"
    echo "= Download the default speeds config ="
    echo "======================================"
    download_file "${default_speeds_config_url}" "${DEFAULT_SPEEDS_CONFIG}"
    updated_default_speed_config=True
  fi
```

Reply:
ah right, let's skip it then